website/source/docs/concepts/tokens.html.md

```diff
@@ -42,14 +42,53 @@ Finally, the `-policy` flag can be used to set the policies associated
 with the token. Learn more about policies on the
 [policies concepts](/docs/concepts/policies.html) page.
 
+<a name="ttls-and-leases" />
 ## Token Time-To-Live and Leases
 
-Every non-root token has a time-to-live (TTL) associated with it.  After the
-TTL is up, the token will no longer function. In addition to no longer
-functioning, Vault will revoke it.
+Every non-root token has a time-to-live (TTL) associated with it, which is
+the number of seconds the token is valid for, _measured from the moment it is
+created_. After the TTL is up, the token will no longer function, and
```

Comment:
Hmmm, so is a new `logical.LeaseOptions` created when you renew? 😕  When I was looking through [`ExtendLease`](https://github.com/hashicorp/vault/blob/master/logical/framework/lease.go), and it looked as though it always measures the TTL from the token's `IssueTime`, rather than the last renewal time...?